autoload/airline.vim

```diff
@@ -71,7 +71,7 @@ endfunction
 
 function! airline#switch_matching_theme()
   if exists('g:colors_name')
-    let existing = g:airline_theme
+    let existing = substitute(g:airline_theme, '-', '_', 'g')
```

Comment:
This doesn't do what you intended it to do. I think what you meant to do was perform this substitution on `g:colors_name`, since vim-airline themes usually use `_` but colorschemes usually use `-`. But instead you're performing it on the `existing` variable, and all that will do is break line 86 where it attempts to switch back to the existing theme (though I don't know why it needs to switch back if it failed to switch away).